src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -31,15 +28,22 @@ import ACHContractStep from './ACHContractStep';
 import EnableStep from './EnableStep';
 import ROUTES from '../../ROUTES';
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
-import reimbursementAccountPropTypes from './reimbursementAccountPropTypes';
+import * as ReimbursementAccountProps from './reimbursementAccountPropTypes';
 import WorkspaceResetBankAccountModal from '../workspace/WorkspaceResetBankAccountModal';
+import reimbursementAccountDraftPropTypes from './ReimbursementAccountDraftPropTypes';
 
 const propTypes = {
     /** Plaid SDK token to use to initialize the widget */
     plaidLinkToken: PropTypes.string,
 
     /** ACH data for the withdrawal account actively being set up */
-    reimbursementAccount: reimbursementAccountPropTypes,
+    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes,
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: reimbursementAccountDraftPropTypes,
+
```

Comment:
Same as above

Reply:
I shared the props for the other ones, but not for this one, as it's the parent that doesn't need **isRequired** on its props.